### ISSUE TITLE
Brugerstyring på søgetemaer

### ISSUE THREAD

**Issue by Ellebjerg** [Septima/spatialsuite-s4#278]
Er det muligt at konfigurere S4, så der kan brugerstyres på søgetemaer?

Vi kan forstå, at styring på endpoints m.m. ikke slår igennem til S4 pga. indekseringen. Kan der laves mulighed for brugerstyring direkte på S4?

**Comment by klavspc** [comment]
Ja, det vil kunne laves. Det vil kræve at vi kender SpS' API således, at vi run-time kan spørge SpS om den aktuelle bruger må til gå en specifik datasource. Det skal nok kunne lade sig gøre, idet vi snakker fint med Sweco. Området er dog nyt for os så jeg kan ikke komme med et bud hvor lang tid det ville tage at implementere i S4.

Husk, at der altid er den simple metode, hvor man laver forskellige tools til forskellige profiler (Hvis man ellers bruger profiler). I S4-toolet står jo hvilke datasources, der søges i - dvs., hvis man har adgang til profilen har man også adgang til søgningen.

**Comment by puwose** [comment]
Vi er også interesseret i Skanderborg Kommune.
Vi har behov for at kunne sætte lås på en S4-datasource, så kun brugere, der har autorisation kan søge i datasourcen.

**Comment by albmorten** [comment]
Albertslund er også interesseret

**Comment by MichaelDavidKnudsen** [comment]
Hej Klavs
Denne er stadig et ønske at få undersøgt.
Kan du ikke give et bud på opgaven, så vi kan komme videre med den?
vh. Michael :-)

**Comment by albmorten** [comment]
Albertslund er også stadig meget interesseret da vores S4 efterhånden er ret fyldt med eget data som kun er relevant for få ansatte pr datakilde.

**Comment by klavspc** [comment]
Vi vil se på det og komme med et bud

@albmorten :
> Albertslund er også stadig meget interesseret da vores S4 efterhånden er ret fyldt med eget data som kun er relevant for få ansatte pr datakilde.

Som jeg læser dette issue så er det rettighedsstyringen, der skal slå igennem i s4 - dvs du vil blive nødt til at rettighedsstyre for at opnå en reduktion i de feature types dine brugere kan se.

--

Fra min egen kommentar længere oppe:
> Husk, at der altid er den simple metode, hvor man laver forskellige tools til forskellige profiler (Hvis man ellers bruger profiler). I S4-toolet står jo hvilke datasources, der søges i - dvs., hvis man har adgang til profilen har man også adgang til søgningen.

**Comment by ChristianHelringAndersen** [comment]
Hej Klavs

hvad er status på denne? :-)

DBH

Christian

**Comment by klavspc** [comment]
Lukket. Dublet af https://github.com/Septima/spatialsuite-s4/issues/366
